control-plane/agents/core/src/controller/wrapper.rs

```diff
@@ -1134,7 +1156,16 @@ impl ClientOps for GrpcClientLocked {
                 Ok(())
             }
             APIVersion::V1 => {
-                unimplemented!()
+                let _ = self
+                    .client_v1()?
+                    .nexus()
+                    .destroy_nexus(v1_conversion::to_rpc(request))
+                    .await
+                    .context(GrpcRequestError {
+                        resource: ResourceKind::Nexus,
+                        request: "destroy_nexus",
+                    })?;
```

Comment:
Same here, if it does not exist it fails on the v1 api.
And probably the other ones have changes as well...